js/src/elements/form-behavior.ts

```diff
@@ -709,9 +719,11 @@ export function navigateAfterSubmit(href: string, navigationStep: NavigationStep
 export function collectCustomElementsByName(form: HTMLFormElement): Element[] {
 	// Compare FormData with form.elements to find missing elements.
 	const formData = Object.fromEntries(new FormData(form));
-	const formElements = Object.keys(form.elements);
 	const customElementList: Element[] = [];
-	const customElements = Object.keys(formData).filter(el => formElements.indexOf(el) === -1);
+	const customElements = Object.keys(formData).filter(el => {
```

Comment:
nit: `el` makes this seem like an element type, but real time is a string, should it be something like `key`, or `name` just to help with readability. Resolve as you please

Reply:
That's true, updated